### PR TITLE
Feature/review card UI

### PR DESCRIPTION
**Pull request by CryoMilo**

### CODE REVIEW

File: apps/web/components/ui/cards/my-review-card.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+						</Stack>
+					</Group>
+
+					<Flex gap="xs" align="center" justify="space-between">
+						<Rating value={courseRating} fractions={2} readOnly />
+						<Flex gap="5px" align="center">
+							<Image alt="uparrowgreen" height={6} src="uparrowgreen.svg" width={4} />
+							30
+							<Image alt="downarrowred" height={6} src="downarrowred.svg" width={4} />
+							40
```
Comment (chinathaip) [inline apps/web/components/ui/cards/my-review-card.tsx:53]: can you use icons as component from [tabler](https://tabler.io/icons) instead?

`<IconArrowUp c="green"/>`
[IconArrowUp](https://tabler.io/icons/icon/arrow-up)

File: apps/web/components/ui/cards/my-review-card.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+					<Group>
+						{limitWords(
+							"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. In pellentesque massa placerat duis ultricies lacus sed.",
+							30
+						)}
+					</Group>
+
+					<Group className="mt-auto w-full" justify="flex-end" gap="xs">
+						<Button bg="blue.4">GO TO MY REVIEW</Button>
```
Comment (chinathaip) [inline apps/web/components/ui/cards/my-review-card.tsx:65]: my bad for this one - kinda regretted it

what do you think about "See my review" or anything more concise

**Review by chinathaip — COMMENTED** [review]
Love the design!

And... we'll need to merge #35  first right?

File: apps/web/components/ui/cards/my-review-card.tsx (hunk 1 above)
Comment (CryoMilo) [reply apps/web/components/ui/cards/my-review-card.tsx:53]: Got it!

**Review by chinathaip — APPROVED** [review]
the card is gucci now. But let's merge this after #35.

@CryoMilo can you start working on the my review page for the time being?

**Comment by chinathaip** [comment]
#35 is merged, can you rebase onto the latest main? @CryoMilo

File: apps/web/utils/limit-text.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export const limitWords = (text: string, limit: number) => {
+	const words = text.split(/\s+/);
+	if (words.length > limit) {
+		return `${words.slice(0, limit).join(" ")}...`;
+	}
+	return text;
+};
```
Comment (Lxkas) [inline apps/web/utils/limit-text.ts:7]: https://tailwindcss.com/docs/text-overflow ?

File: apps/web/app/my-reviews/page.tsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+	return (
+		<Grid
+			className="h-full"
+			classNames={{
+				inner: "h-full"
+			}}
+		>
+			<Grid.Col span={{ base: 12, xl: 12 }}>
+				<Accordion radius="xs" defaultValue="Approved">
```
Comment (chinathaip) [inline apps/web/app/my-reviews/page.tsx:137]:
```suggestion
				<Accordion multiple radius="xs" defaultValue={["Approved"]}>
```

Comment (chinathaip) [reply apps/web/app/my-reviews/page.tsx:137]: I think it's better if we can open multiple items na. Wdyt?

**Comment by chinathaip** [comment]
*Guilty* apologize for the flaky pipeline XD

**Comment by chinathaip** [comment]
done in https://github.com/stamford-syntax-club/course-compose/pull/58

**Pull request closed without merging**